manage-tests/test-reports.mdx

```diff
@@ -0,0 +1,94 @@
+
+Test reports are tied to a single target execution URL, so one specific deployment of your website or app, e.g. a branch deployment during a pull request.
+
+<Frame caption="'Test report page with test report list, 05/2025">
+  <img
+    src="/images/manage/test-report-list.png"
+    alt="Test report page with test report list"
+  />
+</Frame>
+
+## Scheduled test runs
+
+If you want to [run your tests on a fixed schedule](/run-tests/execution-scheduled), the `test reports` section in the `project overview` gives you the option of a daily, weekly or bi-weekly interval.
+
+This ensures that your app works flawlessly over time. We will send you an email notification if any of your tests fail.
+
+<Frame caption="Scheduling test runs in the Octomind app, 05/2025">
+  <img
+    src="/images/run/schedule.png"
+    alt="scheduling test runs in the Octomind app"
+  />
+</Frame>
+
+## Test reports in your CI/CD pipeline
+
+If you integrated Octomind into your [CI pipeline](/integrations/integrations-overview), we will comment test results back in your pull request after a completed test run.
+
+<Frame caption="Example of Octomind test results in a commit comment, 09/2023">
+  <img
+    src="/images/Octomind-test-results-commit-comments-GitHub-example.png"
+    alt="Example of Octomind test results in a github pull request comment"
+  />
+</Frame>
+
+There will be a row for each test with the following information:
+
+| column      | description                                               |
+| ----------- | --------------------------------------------------------- |
+| description | name of the test case                                     |
+| status ✅   | execution successful                                      |
+| status ⌛️   | execution in progress                                     |
+| status ❌   | execution failed                                          |
+| status 💔   | something went wrong on our side during execution         |
+| details     | link which takes you to the test case details (see below) |
+
+## Test report details
+
+When you click on a specific test report in the app or on the link in your commit comment, a `test report overview` with all the results and more details will open.
+
+If trigger manually, you will find the user who triggered it and when. When triggered via CI integration, you will find a link to the `commit` that triggered it.
+
+<Frame caption="Example of a Octomind test report, 05/2025">
+  <img
+    src="/images/manage/test-report.png"
+    alt="Example of a Octomind test report"
+  />
+</Frame>
+
+## Test result details
+
+If you click on one of the tests within the test report, you will find details about the execution and its results.
+
+It will give showcase test step timeline to understand what the execution did exactly. You can swipe through the steps back and forth to see all the interactions Octomind executed for this test run.
```

Comment:
```suggestion
It will show the test step timeline to understand what the execution did exactly. You can swipe through the steps back and forth to see all the interactions Octomind executed for this test run.
```